specs/signature-specification.md

```diff
@@ -209,14 +169,14 @@ A Notation plugin aware implementations MUST use this attribute to verify the si
 The plugin MUST use [Semantic Versioning](https://semver.org/) (SemVer) to use this feature i.e the `get-plugin-metadata` plugin command MUST return a SemVer compliant version in the response.
 A use case for this feature is for a plugin publisher to address security bug in older plugin version, by setting the minimum version to the plugin version with fixes.
 
-See [Guidelines for Notary v2 Implementors](#guidelines-for-notary-v2-implementors) for options to handle these attributes during signature verification.
+See [Guidelines for Notary Project Signature Implementors](#guidelines-for-notary-project-signature-implementors) for options to handle these attributes during signature verification.
 
 ### Unsigned Attributes
 
 These attributes are considered unsigned with respect to the signing key that generates the signature. These attributes are typically signed by a third party (e.g. CA, TSA).
```

Comment:
@toddysm the unsigned attributes mean the attributes are not signed by **the signing key that generates the signature**. The wording `signed` in this sentence `These attributes are typically signed by a third party (e.g. CA, TSA).` referred to the content of the attributes, for example if the content is a certificate, then the certificate itself was signed and issued by a CA.

I would remove this sentence to avoid confusion.